src/Controller/MyController.php

```diff
@@ -13,7 +13,7 @@
 class MyController extends AbstractController
 {
     /**
-     * @Route("/api/v1/swag/my-api-action", name="api.action.swag.my-api-action", methods={"GET"})
+     * @Route("/api/v2/swag/my-api-action", name="api.action.swag.my-api-action", methods={"GET"})
```

Comment:
please do not hardcode the version, use `v{version}` instead. have a look at the core or other plugins for examples :blush: